faunadb/field_test.go

```diff
@@ -69,3 +85,16 @@ func assertFailToExtractField(t *testing.T, value Value, field Field, message st
 	var res Value
 	require.EqualError(t, value.At(field).Get(&res), message)
 }
+
+func assertFailToExtractAnyFieldFor(t *testing.T, values ...Value) {
```

Comment:
It's not being reused. I end up extracting the function just so I could use the varargs to clean up the test a little.